### PR TITLE
Announcing Istio 1.22.6

### PR DESCRIPTION
**Pull request by zirain**
wait https://github.com/istio/release-builder/pull/1958

### CODE REVIEW

File: content/en/news/releases/1.22.x/announcing-1.22.6/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+release: 1.22.6
+---
+
+This release note describes what’s different between Istio 1.22.5 and 1.22.6.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** Support `clusterLocal` host exclusions for multi-cluster.
```
Comment (dhawton) [inline content/en/news/releases/1.22.x/announcing-1.22.6/index.md:16]:
```suggestion
- **Fixed** support for `clusterLocal` host exclusions for multi-cluster.
```

**Review by dhawton — APPROVED** [review]

**Comment by dhawton** [comment]
DNM has long running tests seem to have crashed and are being looked at... make sure to update the date before allowing this to merge.

File: content/en/news/releases/1.22.x/announcing-1.22.6/index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: Announcing Istio 1.22.6
+linktitle: 1.22.6
+subtitle: Patch Release
+description: Istio 1.22.6 patch release.
+publishdate: 2024-10-21
```
Comment (zirain) [inline content/en/news/releases/1.22.x/announcing-1.22.6/index.md:6]: change this before merge

Comment (dhawton) [reply content/en/news/releases/1.22.x/announcing-1.22.6/index.md:6]: See my previous comment.

**Comment by zirain** [comment]
https://github.com/istio/istio/releases/tag/untagged-0907da98700782286648 

@dhawton good to go?

**Pull request merged**